Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/RiskLevelChangeDetector.kt

```diff
@@ -56,7 +57,9 @@ class RiskLevelChangeDetector @Inject constructor(
             Timber.d("Notification Permission = ${notificationManagerCompat.areNotificationsEnabled()}")
 
             if (!foregroundState.isInForeground.first()) {
-                NotificationHelper.sendNotification("", context.getString(R.string.notification_body), true)
```

Comment:
IMHO the if clause could be removed and  `NotificationHelper .sendNotificationIfAppIsNotInForeground` be called instead.

Reply:
Then we depend on static access inside `NotificationHelper` instead of being able to check the injected `foregroundState` in unit tests.

Reply:
In general, it is easier to understand code if methods either check conditions or execute a task, but not a combination